### ISSUE TITLE
"Thou shall" is bad grammar

### ISSUE THREAD

**Issue by ssokolow** [observing/thor#34]
Currently, thor's output contains "Thou shall", which is like writing "he hate" instead of "he hates".

The proper conjugation is "Thou shalt". (Wikipedia's [Thou](https://en.wikipedia.org/wiki/Thou#Conjugation) page does a good job of explaining this in simple terms. Shall is one of the five irregular verbs where you don't just add whichever of -st or -est sounds better.)

**Comment by avishekgurung** [comment]
Your topic of the issue also has a bad grammar. Its "Thou shall" is bad grammar but it should be "Thou shall" is bad grammar

**Comment by ssokolow** [comment]
No, "grammar" is not a countable noun, so it does not directly take an article. (What you're suggesting is equivalent to "a bread".)

**Comment by fikrikarim** [comment]
Good grief..
